pwa/src/apiService/apiService.ts

```diff
@@ -16,7 +16,7 @@ export default class APIService {
 
   public get adminClient (): AxiosInstance {
     return axios.create({
-      baseURL: window.GATSBY_ADMIN_URL,
+      baseURL: process.env.GATSBY_ADMIN_URL,
```

Comment:
Nice. I'm not sure who changed it to `window` but this is how it should be. Nice catch.